e2e/autoscript-apps/src/authn-webauthn/index.html

```diff
@@ -30,6 +30,7 @@
   <body>
     <!-- script src="/_polyfills/fast-text-encoder.js"></script -->
     <button class="login-btn">Login</button>
+    <button class="device-registration">Device Registration</button>
```

Comment:
This is just for that manual test I wrote, has no affect on the automation

Reply:
Yeah I only added it because the only difference is that the previous flow didn't add a device name, but i could just add a device name and remvoe this i think if preferred.

Reply:
The main reason for this flow, is that the tenant i used had a different tree, so I used stoyans tenant and the flow itself requires certs that run without errors.

So my testing was done in a dfiferent env and couldn't use the mock server itself.